src/confdb/db/ConfDB.java

```diff
@@ -1182,25 +1233,27 @@ private void loadConfiguration(Configuration config) throws DatabaseException {
 			psSelectInstances.setInt(8, configId);
 			psSelectInstances.setInt(9, configId);
 			psSelectInstances.setInt(10, configId);
-			rsInstances = psSelectInstances.executeQuery();
+			psSelectInstances.setInt(11, configId); // Tasks
+			psSelectInstances.setInt(12, configId); // Tasks
+			rsInstances = psSelectInstances.executeQuery(); // BSATARIC: get all instances for the same config
+															// (sequences too)
+
+			/*
+			 * if (pera) return;
+			 */
+
 			psSelectPathEntries.setInt(1, configId);
 			psSelectPathEntries.setInt(2, configId);
 			// System.err.println("Trying Pathentries"+configId);
 			rsPathEntries = psSelectPathEntries.executeQuery();
 
-			// This is to fix the operator field bug for modules inside sequences. bug
```

Comment:
what happens tot his part ?

Reply:
You mean why is the code commented below rsPathEntries = psSelectPathEntries.executeQuery(); deleted?

Reply:
Ah, right, I had seen the comment but han't realised the code below it was already commented out.